norse/torch/module/test/test_lif_correlation.py

```diff
@@ -124,5 +124,5 @@ def generate_random_data(
         loss_hist.append(loss_val.item())
         print(f"{e}/{num_episodes}: {loss_val.item()}")
 
-    assert loss_hist[0] > loss_hist[1]
-    assert loss_hist[1] > loss_hist[2]
+    # assert loss_hist[0] > loss_hist[1]
```

Comment:
Remove?

Reply:
I wanted it to leave here as a reminder, there should be a robust way to test it, but currently it randomly fails because the training is not deterministic.